skpro/distributions/base/_base.py

```diff
@@ -49,12 +49,15 @@ class BaseDistribution(BaseObject):
     }
 
     def __init__(self, index=None, columns=None):
-        self.index = _coerce_to_pd_index_or_none(index)
```

Comment:
if would be nice if we could also allow `MultiIndex` in columns - that should not be too much of an additional hassle, should it?